wait-for-idle-runner/Readme.md

```diff
@@ -0,0 +1,15 @@
+# Wait for idle runner GitHub Action
+
+An action that waits for an idle runner.
+The `result` output is set to reflect the state of the runner.
```

Comment:
Sorry to iterate more on this but I think something like this makes a little more sense now that `runner-found` and `runner-idle` are being used:

```suggestion
The `runner-found` output is set to `true` or `false` based on whether or not the runner was found.
If the runner was found, the `runner-idle` output is set to `true` or `false` based on whether the runner is idle.
```